src/app/Events/Updated.php

```diff
@@ -94,7 +94,7 @@ private function readRelation($relation, $value)
         $class = key($relation->toArray());
         $attribute = $relation->get($class);
 
-        return $class::find($value)->{$attribute};
+        return optional($class::find($value))->{$attribute};
```

Comment:
why do we need optional? Are you considering relations on other attributes than ids?

Reply:
if the relation changed from null to something or from something to null this line produce an error.